keps/sig-node/2008-forensic-container-checkpointing/README.md

```diff
@@ -292,14 +329,94 @@ Checkpointing containers will be possible again.
 
 ###### Are there any tests for feature enablement/disablement?
 
-Currently no.
+Currently the test will automatically be skipped if the feature is not enabled.
+
+### Rollout, Upgrade and Rollback Planning
+
+Does not apply as the feature is an additional API endpoint with no
+dependencies on other functionality. If it is not enabled via the feature
+gate it will return `404 page not found`. If it is not enabled in the
+underlying container engine a `500` will be returned with an error
+message from the container engine. If it is enabled the API endpoint exists
+if disabled then it does not exist. No planning necessary.
+
+Documented at <https://kubernetes.io/docs/reference/node/kubelet-checkpoint-api/>
+<!--
+This section must be completed when targeting beta to a release.
+-->
+
+###### How can a rollout or rollback fail? Can it impact already running workloads?
+
+At this point it is still a kubelet only API endpoint and has no dependencies
+on other components.
+
+###### What specific metrics should inform a rollback?
+
+The only metric is the return code from the API endpoint.
+
+###### Were upgrade and rollback tested? Was the upgrade->downgrade->upgrade path tested?
+
+No, this does not seem to apply for this feature.
```

Comment:
Explain why not.  You could probably justify it as, "no data is stored, so re-enabling starts from a clean slate"

Reply:
Will do.

Reply:
This is a kubelet only feature so..